From 226ce3d104f9be48c5ec70794639dade0662d0ab Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Fri, 26 Feb 2016 15:45:30 +0000
Subject: [PATCH] udev rules: add kvm rule

Based on commit e8c942d0df396e3c756a7261a8bffcb8c9847a1a.

Make /dev/kvm available to members of the kvm group. This improves
security since it avoids having to be root to make use of kvm.

Signed-off-by: Vu Tran <vu.tran@windriver.com>
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 rules/99-kvm.rules | 1 +
 rules/meson.build  | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 rules/99-kvm.rules

diff --git a/rules/99-kvm.rules b/rules/99-kvm.rules
new file mode 100644
index 0000000..c2f7317
--- /dev/null
+++ b/rules/99-kvm.rules
@@ -0,0 +1 @@
+KERNEL=="kvm", GROUP="kvm", MODE="0660"
diff --git a/rules/meson.build b/rules/meson.build
index e253b9f..6c74b73 100644
--- a/rules/meson.build
+++ b/rules/meson.build
@@ -36,6 +36,7 @@ rules = files('''
         78-sound-card.rules
         80-drivers.rules
         80-net-setup-link.rules
+        99-kvm.rules
 '''.split())
 
 install_data(rules,
-- 
2.7.4

